src/__stories__/Customization.stories.mdx

```diff
@@ -1,12 +1,18 @@
-import { Meta, Story } from '@storybook/addon-docs/blocks'
-import { colors } from '..'
-import Box from '../..'
+import { Meta } from '@storybook/addon-docs/blocks'
+import { Box } from '..'
+import { colors } from '../theme'
 
-<Meta title="Theme/Colors" />
+<Meta title="Customization" />
```

Comment:
Is there a reason to call this page customization ? Can't we call it Theme instead ?
Later if we want to add more documentation about customization we will create an advanced category with sub documentation pages but from now I think Theme as title will be enough 😄